libvast/include/vast/query_context.hpp

```diff
@@ -144,13 +144,12 @@ struct query_context {
   vast::ids ids = {};
 
   struct priority {
-    constexpr static uint8_t high = 100;
-    constexpr static uint8_t normal = 25;
-    constexpr static uint8_t low = 1;
+    constexpr static uint64_t normal = 1'000;
+    constexpr static uint64_t low = 1;
   };
 
   /// The query priority.
-  uint8_t priority = priority::normal;
+  uint64_t priority = priority::normal;
```

Comment:
We sum up query priorities to get a partition priority for the queue. Restricting this to 0-255 was quite intentional. Can we try to find better defaults with that in mind?

Reply:
I think this is a minor issue in code that is bound to disappear entirely with `from vast`, so I think for now this is absolutely alright to leave as-is.

Reply:
Alright, since will be replaced in the foreseeable future...